docs/source/core_developers_guide.rst

```diff
@@ -0,0 +1,12 @@
+
+**********************
+Core Developer's Guide
+**********************
+
+.. toctree::
+   :maxdepth: 2
+
+   core_developers_guide/sawtooth_vagrant
+   core_developers_guide/intkey_workload.rst
```

Comment:
The .rst is optional apparently, and was from a previous commit not in this PR.  I've added a commit to remove it as stylistically we should be consistent.